subsys/bluetooth/host/id.c

```diff
@@ -1281,20 +1281,27 @@ int bt_id_create(bt_addr_le_t *addr, uint8_t *irk)
 {
```

Comment:
You're changing behavior.
Please update [id_create's](https://github.com/jori-nordic/zephyr/blob/49f1414d9dc74eb2fc3e23fc6ec4d23efffb4108/include/zephyr/bluetooth/bluetooth.h#L359) documentation.